components/prize_pool/commons/prize_pool_import.lua

```diff
@@ -328,10 +328,11 @@ function Import._computeBracketPlacementGroups(bracket, options)
 
 			-- Opponents knocked out from sole section (se) or lower bracket (de)
 			if (not bracket.bracketDatasById[matchId].qualLose or options.importWinners) and (
```

Comment:
imo the if conditions get messy here
maybe make it a function so we can "just"
`if Import_someFunctionName(bracket, matchId, options, coordinates, firstDropdownRoundIndexes) then`